apps/mail/components/mail/mail-list.tsx

```
@@ -817,10 +813,25 @@ export const MailList = memo(
             ) : (
               <div className="flex flex-1 flex-col" id="mail-list-scroll">
                 <VList
+                  ref={vListRef}
                   count={filteredItems.length}
                   overscan={5}
                   className="style-scrollbar flex-1 overflow-x-hidden"
                   children={vListRenderer}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)